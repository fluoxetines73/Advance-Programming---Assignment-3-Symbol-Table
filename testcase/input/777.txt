INSERT d string
RPRINT
BEGIN
RPRINT
LOOKUP d
ASSIGN d 'abcABC'
BEGIN
LOOKUP d
PRINT
LOOKUP d
INSERT d number
INSERT f_FF number
PRINT
PRINT
ASSIGN f_FF 987654
INSERT a string
INSERT x number
RPRINT
RPRINT
END
END