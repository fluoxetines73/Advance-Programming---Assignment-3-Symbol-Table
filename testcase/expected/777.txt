success
d//0
d//0
0
success
0
d//0
0
success
success
d//2 f_FF//2
d//2 f_FF//2
success
success
success
x//2 a//2 f_FF//2 d//2
x//2 a//2 f_FF//2 d//2